cfgov/core/forms.py

```diff
@@ -30,7 +32,7 @@ def clean(self):
         if matched_whitelist:
             cleaned_data['validated_url'] = url
 
-        elif 'signature' in cleaned_data:
+        elif cleaned_data.get('signature'):
```

Comment:
@rossk this change required because `'signature'` will always exist as the empty string in `cleaned_data` due to how the form is defined.